runtime/ftplugin/man.vim

```diff
@@ -22,7 +22,8 @@ setlocal tagfunc=man#goto_tag
 if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
   nnoremap <silent> <buffer> j             gj
   nnoremap <silent> <buffer> k             gk
-  nnoremap <silent> <buffer> gO            :call man#show_toc()<CR>
+  nnoremap <silent> <buffer> gF            :call man#show_flags()<CR>
```

Comment:
I think `gF` might not be ideal, since it's a default mapping for navigation to a filename under cursor and in man pages is usual to find said filenames, I personally navigate often to the listed ones in manpages.

I personally prefer to see the flags as subcategories in the TOC, or in a mapping like `[count]gO`.

Reply:
> I personally prefer to see the flags as subcategories in the TOC

I like this too. Just added the change.